### PR TITLE
Show mastodon on modal

### LINKED ISSUE #6119: /share is too blank
We should have the mastodon at the bottom of the drawer column.
https://github.com/tootsuite/mastodon/blob/master/app/javascript/images/mastodon-drawer.png

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

### PR DESCRIPTION
**Pull request by akihikodaki**
screenshot-2017-12-29 mastodon dev 4
Resolves #6119.

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 !!! 5
-%html{ lang: I18n.locale }
+%html{ class: yield(:html_classes), lang: I18n.locale }
```
Comment (ykzts) [inline app/views/layouts/application.html.haml]: Is this change necessary?

**Comment by akihikodaki** [comment]
97fd0b7c22218ebc2b6b628494f8b2e3ab126899: removed an irrelevant change pointed out in https://github.com/tootsuite/mastodon/pull/6129#pullrequestreview-86056887.

**Review by ykzts — APPROVED** [review]
:heart:
